frontend/packages/console-app/src/components/nodes/csr.ts

```diff
@@ -53,19 +53,19 @@ export const getNodeClientCSRs = (
         metadata: {
           ...csr.metadata,
           name: commonName.value.valueBlock.value.replace('system:node:', ''),
-          originalName: csr.metadata.name,
+          originalName: csr.metadata?.name ?? '',
         },
       };
     })
     .sort(
       (a, b) =>
-        new Date(b.metadata.creationTimestamp).getTime() -
-        new Date(a.metadata.creationTimestamp).getTime(),
+        new Date(b.metadata?.creationTimestamp ?? '').getTime() -
+        new Date(a.metadata?.creationTimestamp ?? '').getTime(),
```

Comment:
this may cause a runtime error:

```
> new Date('')
Invalid Date

> (new Date('')).getTime()
NaN 
```

please double check if this works fine if creationTimestamp is null